### ISSUE TITLE
Breadcrumbs triggering accessibility rule on Lighthouse

### ISSUE THREAD

**Issue by ptdev** [bootstrap-vue/bootstrap-vue#2988]
Hi, the breadcrumb component appears to fail [the following Lighthouse accessibility rule](https://dequeuniversity.com/rules/axe/3.1/list):

```
<ul> and <ol> must only directly contain <li>, <script> or <template> elements
```

To test, simply run Lighthouse on the breadcrumb documentation page or any other place that's using it.

Cheers

**Comment by tmorehouse** [comment]
The structure should be `ol > li > a|span` and shouldn't contain any other elements. Unless is is complaining about the psuedo elements used as separators. 

Do you see the same issue on the getbootstrap.com docs site?

**Comment by ptdev** [comment]
Hi,
I'm actually not getting it on getbootstrap.com website, which is strange imho because I am getting it on my current development as well as bootstrap-vue's docs website.

On my website, using the b-breadcrumb component yields the following markup:
```
<ol class="breadcrumb">
    <li role="presentation" class="breadcrumb-item">
        <a href="/" target="_self" class="nuxt-link-active">Home</a>
    </li>
    <li role="presentation" class="breadcrumb-item active">
        <span aria-current="location">Contacts</span>
    </li>
</ol>
```

On bootstrap-vue docs
```
<ol class="breadcrumb">
    <li role="presentation" class="breadcrumb-item">
        <a target="_self" href="#" class="">Admin</a>
    </li>
    <li role="presentation" class="breadcrumb-item">
        <a target="_self" href="#" class="">Manage</a>
    </li>
    <li role="presentation" class="breadcrumb-item active">
        <span aria-current="location">Library</span>
    </li>
</ol>
```

On getbootstrap docs:
```
<ol class="breadcrumb">
    <li class="breadcrumb-item">
        <a href="#">Home</a>
    </li>
    <li class="breadcrumb-item">
        <a href="#">Library</a>
    </li>
    <li class="breadcrumb-item active" aria-current="page">
        Data
    </li>
</ol>
```

As far as I can see, they all appear identical (?)
Can you pelase confirm that if you run lighthouse on your end you're also getting this?

**Comment by tmorehouse** [comment]
The only difference is the `presentation` role on the rendered `li` elements.

Maybe they used to have that role at bootstrap but have since removed it,  and we haven't updated to match.

**Comment by ptdev** [comment]
There's another difference in that the getbootstrap version doesn't have any span elements inside, which, afaik, is what that rule is complaining about.
This is not really critical for me, but just more of an annoyance if one's aiming for that perfect lighthouse score.

**Comment by tmorehouse** [comment]
Almost any element (inline block or block) is allowed inside `li` elements. I think since the role has been set as presentation it might be complaining that the li's are no longer semantically an li

**Comment by ptdev** [comment]
Hey, 
You are correct. I just ran a test with just the breadcrumb markup and removing role="presentation" does indeed get rid of the rule alert.

**Comment by tmorehouse** [comment]
Ok... we'll update on our end here.

**Comment by jackmu95** [comment]
@tmorehouse I've looked all the way back to the Bootstrap v4.0 docs and never found `role="presentation"` used.

**Comment by tmorehouse** [comment]
It might have ben from the alpha days of bootstrap 4
